plugins/notifiers/slack/client_test.go

```diff
@@ -0,0 +1,77 @@
+package slack
```

Comment:
Let's use test package name like the others

```suggestion
package slack_test
```

Reply:
Will take these test package changes in a separate PR, since it would need some refactoring in the `slack/client.go` file to make `notifier` struct public.